Kaspersky/kaspersky_endpoint_security/tests/test_not_processed.json

```
@@ -0,0 +1,42 @@
+{
+  "input": {
+    "sekoiaio": {
+      "intake": {
+        "dialect": "Kaspersky Endpoint Security",
+        "dialect_uuid": "255764ef-eaf6-4964-958e-81b9418e6584"
+      }
+    },
+    "message": "Event type: Object not processed\\r\\nName: msiexec.exe\\r\\nApplication path: C:\\Windows\\System32\\r\\nProcess ID: 7684\\r\\nUser: WORKGROUP\\MyMachine$ (Initiator)\\r\\nComponent: File Threat Protection\\r\\nResult description: Not processed\\r\\nObject type: File\\r\\nPath to object: C:\\Windows\\Installer\\r\\nObject name: 8056b1f.msi\\r\\nReason: Size"
+  },
+  "expected": {
+    "message": "Event type: Object not processed\\r\\nName: msiexec.exe\\r\\nApplication path: C:\\Windows\\System32\\r\\nProcess ID: 7684\\r\\nUser: WORKGROUP\\MyMachine$ (Initiator)\\r\\nComponent: File Threat Protection\\r\\nResult description: Not processed\\r\\nObject type: File\\r\\nPath to object: C:\\Windows\\Installer\\r\\nObject name: 8056b1f.msi\\r\\nReason: Size",
+    "event": {
+      "action": "Not processed",
+      "category": ["process"],
+      "kind": "event",
+      "module": "File Threat Protection",
+      "reason": "Object not processed",
```

Maybe we can create something like that ?

```suggestion
      "reason": "Object not processed Reason: Size",
```

---

@Adamowoc what do you think about `Object not processed : Size` or `Object not processed because of Size` ?